cfpb/development
# Rename pages as part of information architecture changes

Rename pages (either in the main navigation from the README, in their page titles, or both) to more closely match the content that's on them

## Changes

- Rename pages with titles that we identified as unclear
- Rename all the language standards pages to "{Language} Standards" so that they're consistent with the README navigation and with each other

## Testing

1. View this branch in GitHub
2. Make sure all the new page and navigation titles make sense
